src/components/views/game/game/GameView.tsx

```diff
@@ -1,34 +1,24 @@
 import React from "react"
-import { useNavigate } from "react-router-dom"
-import { Paths } from "../../../routing/routers/Paths"
-import { useCurrentGame } from "../../../../hooks/api/useCurrentGame"
 import { observer } from "mobx-react-lite"
 import { GameTable } from "./gametable/GameTable"
 import { VideoChat } from "./videochat/VideoChat"
 import { ScoreBoard } from "./scoreboard/ScoreBoard"
-import { Login } from "@mui/icons-material"
-import { Button, Grid } from "@mui/material"
+import { Grid } from "@mui/material"
 import { YourHand } from "./yourhand/YourHand"
 import "./GameView.scss"
 import ScoreAnnouncing from "./scoreannouncing/ScoreAnnouncing"
+import HamburgerMenu from "../../../ui/HamburgerMenu"
 
 /**
  * The main game page inside the Jass-Stube includes a table with all players
  * @param props
  * @returns
  */
 const GameView = observer(() => {
-  const navigate = useNavigate()
-  const { loading, leaveGame } = useCurrentGame()
-
-  const clickLeave = async () => {
-    await leaveGame()
-    navigate(Paths.LOBBY)
-  }
-
   return (
     <div className="game div-box">
       <Grid container spacing={0} className={"top-row"}>
+        <HamburgerMenu />
         <Grid item xs={1} />
```

Comment:
```suggestion
        <Grid item xs={1} >
            <HamburgerMenu />
        </Grid>
```
Meine Hoffnung war, dass das grid die positionierung übernehmen kann.

Reply:
Ich hab mal versucht einfach das so zu machen, wie den Videochat und das Scoreboard - Weil sonst verschiebt sich alles irgendwie